docs/pages/development/installation.md

```diff
@@ -68,15 +68,19 @@ No additional changes are needed to install the package on Android. 🎉
 
 ## 2. Basic configuration
 
-Development builds use deep links to open projects from the QR code. If you have added a custom deep link scheme to your project, your development build will use it. However, if this isn't the case, you need to configure the deep link support for your app. The `uri-scheme` package will do this for you once you have chosen a scheme.
+Development builds use deep links to open projects via a QR code. If you have added a custom deep link scheme to your project, your development build will use it. However, if this isn't the case, you need to configure deep link support for your app. The `uri-scheme` package will do this for you, once you have chosen a scheme. 
+
+A scheme simply tells your device's operating system which app to open based on the part of a URL preceeding the colon. For example, in `exp://192.168.4.54:19000`, `exp` is the scheme and your device will leave it to Expo Go (the app matching the `exp` scheme) to interpret `192.168.4.54:19000`, assuming everything is properly configured.
+
+If you haven't already done so, make up your own scheme and run the following command to configure deep link support for your app.
 
 <InstallSection packageName="expo-development-client" cmd={["npx uri-scheme add <your scheme>"]} hideBareInstructions />
 
 See the [uri-scheme package](https://www.npmjs.com/package/uri-scheme) for more information.
 
 ### 🍏 iOS
```

Comment:
Can you please remove these emojis from headers (I know they are unrelated to your PR) and also the "🎉 " ones 🙇